### DISCUSSION TITLE
If Nemo uses langchain under the cover, is there any need to use langchain then?

### DISCUSSION THREAD

**Discussion by sidpotti** [NVIDIA-NeMo/Guardrails#80] [Q&A]
If Nemo uses langchain under the cover, is there any need to use langchain then? Or should langchain still be used to specify specific features of the LLM?

**Comment by drazvan** [comment]
It depends on the use case. If you need to write some custom actions that also invoke the LLM, you can use langchain to write that logic. Is there something specific that you're thinking about in terms of functionality?
